Assets/Scripts/Models/GameModel.cs

```diff
@@ -0,0 +1,55 @@
+using UnityEngine;
+using Player;
+using Weapon;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Game {
+  public class GameModel {
+
+    public static int MAX_PLAYER_CAPCITY = 100;
+
+    private ArrayList Players = new ArrayList();
```

Comment:
Let's make this type explicit. We don't want variables of other types to be added.

Reply:
I'll try to figure out how can I do this. Because initially I tried to make it explicit but there's compile error